### ISSUE TITLE
Improve printing

### ISSUE THREAD

**Issue by jd-lara** [JuliaStochOpt/ParameterJuMP.jl#36]
The current printing of a constraint with params can be confusing since it adds a 0.0 on the RHS. 

```Julia
julia> m = ModelWithParams();
julia> a = Parameter(m, 10);
julia> x = @variable(m);
julia> t = @constraint(m, x <= a)
noname ≤ 0.0
```

**Comment by joaquimg** [comment]
Uhm, thats because the model is only synced once all variables are added.
Maybe we can add a mode to passa the constants directly.
Having parameters themselves in the printing might be harder.

**Comment by jd-lara** [comment]
Agreed, this seems to be a hard problem but it's worth registering it into the issues for reference.

**Comment by joaquimg** [comment]
One option would be to create a ParametricConstraintRef type.
